provider/kubernetes/kubernetes_test.go

```diff
@@ -1109,6 +1109,15 @@ func TestIngressClassAnnotation(t *testing.T) {
 					iPaths(onePath(iPath("/derp"), iBackend("service1", intstr.FromInt(80))))),
 			),
 		),
+		buildIngress(
+			iNamespace("testing"),
+			iAnnotation(annotationKubernetesIngressClass, "custom"),
+			iRules(
+				iRule(
+					iHost("herp"),
```

Comment:
Could you replace the `herp/derp` with `foo/bar` or `foo/path` to match the format of other tests?